docs/configuration/index.md

```diff
@@ -367,23 +370,31 @@ The following monitors are available:
 |`org.apache.druid.server.emitter.HttpEmittingMonitor`|Reports internal metrics of `http` or `parametrized` emitter (see below). Must not be used with another emitter type. See the description of the metrics here: https://github.com/apache/druid/pull/4973.|
 |`org.apache.druid.server.metrics.TaskCountStatsMonitor`|Reports how many ingestion tasks are currently running/pending/waiting and also the number of successful/failed tasks per emission period.|
 
+> Metric monitoring is an essential part of Druid operations.  For example, you might configure monitors on all processes for system and JVM information as follows:
```

Comment:
Remove from admonition `>` . Make this the introduction to this section before "The following monitors are available". Line 356.